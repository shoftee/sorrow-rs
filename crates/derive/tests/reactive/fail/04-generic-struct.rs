use sorrow_derive::Reactive;

#[derive(Reactive)]
pub struct Generic<T> {
    inner: T,
}

fn main() {}
