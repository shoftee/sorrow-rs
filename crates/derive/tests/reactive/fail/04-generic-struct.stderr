error: derive(Reactive) does not support generics.
 --> tests/reactive/fail/04-generic-struct.rs:4:19
  |
4 | pub struct Generic<T> {
  |                   ^^^
